docs/terminals/overview.mdx

```diff
@@ -0,0 +1,123 @@
+and exceptional customer experiences.
+
+Our unified card present solution brings both card not present and card present transactions 
+into a single system. This not only simplifies processing but also streamlines reporting and support. 
+By doing so, we offer platforms a smoother, more intuitive user experience while also making 
+it easier to gain valuable insights and resolve issues. 
+It's a win-win situation for platforms and their customers alike.
+
+## Terminal Models
+We offer a variety of terminal models. Here are the most commonly used ones:
+
+### e285 - Companion Device or 3G Standalone 
+Standalone mobile point of sale device with WiFi, Bluetooth and 3G communication options, 
+allowing secure payment acceptance anywhere without dependence on a smart device. 
+The solution features a color touch screen capable of signature capture and is built on the Verifone Engage platform,
+enabling rich customer interaction at the point of sale.
+
+<img style={{float: 'right'}}
+  src={TerminalE285}
+  alt="terminal model e285"
+  object-fit="contain"
+  width="43%"
+  height="43%"
+/>
+
+**Specs**\
+2.8” Touch Display\
+3G / DB WiFi / BT\
+**Wired:** USB-C\
+**All Card** Acceptance\
+**Printer:** None\
+**Keypad:** Mechanical\
+**Battery:** 3.8V 1800mAh\
+**Scanning:** Companion Device\
+**ENV:** Indoor mobile\
+**Memory:** 1024 MB\
+**OS:** V/OS2+ADK
+
+
+### P400 - Integrated; Food & Beverage, Retail, Supermarket
+Advanced, high performance pin pad with rich multimedia and commerce capabilities. 
+The P400 is a consumer-facing handheld device. It can also be fix-mounted in some 
+integrated retail scenarios. The product’s design is equally appealing as a handheld 
+PINpad and robust enough to look and function appropriately in a fixed mount setting.	
+
+<img style={{float: 'right'}}
+  src={TerminalP400}
+  alt="terminal model P400"
+  object-fit="contain"
+  width="43%"
+  height="43%"
+/>
+
+**Specs**\
+3.5” Touch Display\
+**Wired:** Ethernet, USB, RS232\
+**All Card** Acceptance\
+**Printer:** None\
+**Keypad:** Mechanical\
+**Audio:** Buzzer\
+**Battery:** None\
+**Scanning:** None\
+**ENV:** Indoor Countertop\
+**Memory:** 384 MB, micro SD\
+**OS:** V/OS2+ADK\
+
+
+### V400m - Hospitality, Outdoor Market, Transportation, Retail
+A high-end, touchscreen countertop solution. Includes power supply and paper roll in the box.
+
+<img style={{float: 'right'}}
+  src={TerminalV400m}
+  alt="terminal model V400m"
+  object-fit="contain"
+  width="43%"
+  height="43%"
+/>
+
+**Specs**\
+3.5” Touch Display\
+4G / Dual SIM\
+**Wired:** USB-C\
+**All Card** Acceptance\
+**Printer:** Thermal, 40mm roll\
+**Keypad:** Mechanical\
+**Audio:** Speaker\
+**Power:** 5V PSU\
+**Battery:** 3.8V 2890mAh\
+**ENV:** Indoor Portable, IP52\
+**Memory:** 1024 MB\
+**OS:** V/OS2+ADK\
+
+## Ready to Get Started?
+[Get in touch with us](https://docs.justifi.tech/contact) and learn all the ins and outs about the JustiFi Unified Card Present program.
+
+<!--
```

Comment:
Same here. Will need to add a trouble shooting section but not sure yet where it should go and what it should include. Ideas are welcome ;)